example/example.cpp

```diff
@@ -1,33 +1,75 @@
 #include <capo/capo.hpp>
 #include <ktl/str_format.hpp>
 #include <chrono>
+#include <cmath>
 #include <fstream>
 #include <iostream>
 #include <thread>
 
 namespace impl {
 static constexpr int fail_code = 2;
+// TODO: Add parameters for each one of these constants [Use clap?]
+static constexpr float travel_circurference_radius = 2;
+static constexpr float travel_angular_speed = 2;
+static constexpr bool loop_audio = true;
 
-int openAlTest(std::string const& wavPath, float gain) {
-	auto pcm = capo::PCM::fromFile(wavPath);
-	if (!pcm) { return impl::fail_code; }
+// Uniform circular motion formulas
+capo::Vec3 UCMPosition(float angularSpeed, float time, float radius) {
```

Comment:
Is this going to be useful in a game engine?

Reply:
No, it's just an example position and velocity function

Reply:
Whoops, my bad, I didn't notice the file. 😅